### ISSUE TITLE
M3U Output?

### ISSUE THREAD

**Issue by jimmisavage** [tombowditch/telly#296]
Is there a way to pull an M3U from Telly to use on another device/app, such as TiviMate or Nextpvr? Plex is fine but it doesn't perform as well as TiviMate for IPTV.

**Comment by chazlarson** [comment]
No.  There are plenty of purpose-built m3u filtering solutions already.

**Comment by jimmisavage** [comment]
I see. That's a shame.
I've been looking for something that basically changes the stream URL, or passes it through a VPN as my ISP blocks my IPTV. I was hoping telly might have been a solution but i guess i've miss-understood.

**Comment by chazlarson** [comment]
Telly reads/filters the M3U then presents a fake HD Homerun interface for Plex to connect to.  That's all it's intended to do.  It does proxy the stream if you're transcoding it with ffmpeg, but that's done through that virtual HD Homerun.

I honestly can't recall if the stream URLs presented in the virtual HD HOmerun when ffmpeg is turned ON point to the telly machine; they probably do.  In that case you could create a thin proxy that converted the virtual HD Homerun channel listing back into an m3u.
